control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -232,6 +232,7 @@ func ReconcileRole(role *rbacv1.Role, ownerRef config.OwnerRef, platform hyperv1
 			},
 		}...)
 	}
+	// TODO (jparrill): Add RBAC specific needs for Agent platform
```

Comment:
how is this related to this PR?

Reply:
I've seen this section when I was reviewing the new HCCO cert and account, and this is something I need add for the agent platform, not 100% related.